src/api/todos.ts

```diff
@@ -5,4 +5,22 @@ export const getTodos = (userId: number) => {
   return client.get<Todo[]>(`/todos?userId=${userId}`);
 };
 
-// Add more methods here
+export const createTodo = (
+  title: string,
+  userId: number,
+  completed: boolean,
```

Comment:
It's better to pass object, so you will never make mistake with wrong arguments order

Reply:
mb, but i dont wanna create new interface for object without id.